draft-ietf-tls-tls13.md

```diff
@@ -2043,21 +2043,19 @@ The "extension_data" field of this extension contains a
 
 %%% Key Exchange Messages
 
-       enum { psk_ke(0), psk_dhe_ke(1), (255) } PskKeyExchangeModes;
-       enum { psk_auth(0), psk_sign_auth(1), (255) } PskAuthenticationModes;
-
-       opaque psk_identity<0..2^16-1>;
+       enum { psk_ke(0), psk_dhe_ke(1), (255) } PskKeyExchangeMode;
+       enum { psk_auth(0), psk_sign_auth(1), (255) } PskAuthenticationMode;
 
        struct {
-           PskKeMode ke_modes<1..255>;
-           PskAuthMode auth_modes<1..255>;
+           PskKeyExchangeMode ke_modes<1..255>;
+           PskAuthenticationMode auth_modes<1..255>;
            opaque identity<0..2^16-1>;
        } PskIdentity;
 
        struct {
            select (Role) {
                case client:
-                   psk_identity identities<2..2^16-1>;
+                   PskIdentity identities<4..2^16-1>;
```

Comment:
Hmm, looks like the minimum length is 6, though (two nonempty 1-byte-length fields at 2 bytes each, plus two bytes of length for the optional identity field).